src/deploy/scope_managementgroup.ts

```diff
@@ -68,7 +67,11 @@ export async function DeployManagementGroupScope(azPath: string, region: string,
         core.info("Creating deployment...")
         var deploymentCode = await exec(`"${azPath}" deployment mg create ${azDeployParameters} -o json`, [], deployOptions);
         if (commandStdErr.trim().length !== 0) {
-            throw new Error(`Deployment process failed as some lines were written to stderr: ${commandStdErr}`)
+            if (failOnStdError.toLowerCase().trim() == "true") {
```

Comment:
Pass failOnStdError as a boolean here. To the string trim and comparison in entry point only